### DISCUSSION TITLE
Should all OBA terms end with "trait" or not?

### DISCUSSION THREAD

**Discussion by matentzn** [obophenotype/bio-attribute-ontology#110] [Q&A]
Currently, there is a bit of a mix of terms ending in "trait" or not. In our last call, we proposed that we consistently add the word "trait" to every label in OBA. So, rather than "ulna size", we say "ulna size trait", and have "ulna size" as an exact synonym.

For some people like sounds redundant, for sure, but the label also helps during curation scenarios to ensure that OBA traits are not accidentally used as "assay" or "measurement" terms. (ulna size measurement here denoting the process of measuring the size of the ulna).

So, we propose to consistently use the word "trait" at the end of all OBA terms. Objections?

**Comment by cmungall** [comment]
Seems ok for grouping classes but odd for specific attributes

Exercise increases my heart rate trait

Drugs to lower blood pressure trait

Regulation of cell volume trait

**Comment by meghalithic** [comment]
the alternative is to remove "trait", so that we'd have terms like "ulna size" or "radius bone size". I proposed to add trait originally because for the bones size traits there were fewer terms with "trait" than without "trait", I was not thinking about other terms that are traits.

**Comment by cmungall** [comment]
Size is odd because it means two things - a grouping for specific
dimensions (bone length is a size trait, increase in size may mean increase
in width only) and a specific quality (all dimensions combined)

I think using trait for groupings is fine but it seems odd on specific
qualities
